draft-ietf-scitt-architecture.md

```diff
@@ -149,17 +149,17 @@ Issuers can register their Signed Statements on any Transparency Service, with t
 This document describes the scalable, flexible, and decentralized SCITT architecture.
 Its goal is to enhance auditability and accountability across supply chains.
 
-In supply chains, artifacts travel down the chain until they are eventually consumed by someone.
-Consumers like to have information about the artifacts that they consume.
+In supply chains, artifacts travel down the chain until they are no longer usable for one or more parties.
+All parties in supply chains like information about these artifacts.
```

Comment:
This sentence is a throwaway, better to delete it

Reply:
I am fine with that.